Makefile

```diff
@@ -17,3 +17,12 @@ phpcs:
 covers:
 	./vendor/bin/covers-validator
 
+# MediaWiki Core related
```

Comment:
All three of these should have been added to `.PHONY`, no? (GitHub won’t let me comment on that line.)